python/cuspatial/cuspatial/core/binpreds/feature_contains.py

```diff
@@ -49,7 +49,23 @@ def _preprocess(self, lhs, rhs):
         preprocessor_result = super()._preprocess_multipoint_rhs(lhs, rhs)
         return self._compute_predicate(lhs, rhs, preprocessor_result)
 
-    def _intersection_results_for_contains(self, lhs, rhs):
+    def _intersection_results_for_contains_linestring(self, lhs, rhs):
```

Comment:
The name sounds abstruse. It counts the number of intersecting points (and the segment vertices) that are equal to the rhs vertices. Perhaps `_count_equal_intersecting_points`?

Reply:
I also believe function names should reflect its implementation, not purpose.